modules/driver-mongo/src/main/kotlin/datamaintain/db/driver/mongo/MongoDriver.kt

```diff
@@ -52,10 +53,26 @@ class MongoDriver(dbName: String,
     }
 
     override fun listExecutedScripts(): Sequence<Script> {
-        return executedScriptsCollection.find().asSequence()
+        return executedScriptsCollection.find().asSequence().map { documentToScriptWithoutContent(it) }
     }
 
     override fun markAsExecuted(script: Script) {
-        executedScriptsCollection.insertOne(ScriptWithoutContent(script.name, script.checksum, script.identifier))
+        val scriptWithoutContent = ScriptWithoutContent(script.name, script.checksum, script.identifier)
+        val scriptWithoutContentDocument = scriptWithoutContentToDocument(scriptWithoutContent)
+        executedScriptsCollection.insertOne(scriptWithoutContentDocument)
+    }
+
+    fun scriptWithoutContentToDocument(script: ScriptWithoutContent): Document {
+        return Document()
+                .append("name", script.name)
+                .append("checksum", script.checksum)
+                .append("identifier", script.identifier)
+    }
+
+    fun documentToScriptWithoutContent(document: Document): ScriptWithoutContent {
+        val name: String = document.getString("name")
+        val checksum: String = document.getString("checksum")
+        val identifier: String = document.getString("identifier")
+        return ScriptWithoutContent(name, checksum, identifier)
```

Comment:
You should extract ```"name"```, ```"checksum"``` and ```"identifier"``` to global final static variables to avoid the risk of editing one string without editing the other one and cause bugs in serialization.

Reply:
I agree with @Lysoun